src/manage_sql_tls_certificates.c

```diff
@@ -180,6 +180,45 @@ tls_certificate_select_columns ()
   return columns;
 }
 
+/**
+ * @brief Get extra_where string for a TLS certificate iterator or count.
+ *
+ * @param[in]  filter           Filter string.
+ *
+ * @return     Newly allocated extra_where string.
+ */
+gchar *
+tls_certificate_extra_where (const char *filter)
+{
+  GString *ret;
+  gchar *report_id;
+
+  ret = g_string_new ("");
+
+  report_id = filter_term_value (filter, "report_id");
+
+  g_message ("report_id=\"%s\"", report_id);
+
+  if (report_id)
+    {
+      gchar *quoted_id;
+      quoted_id = sql_quote (report_id);
+      g_string_append_printf (ret,
+                              " AND"
+                              " (EXISTS (SELECT * FROM"
+                              "  tls_certificate_source_origins AS src_orig"
```

Comment:
I know it's a small detail, but I'd really like to keep SQL statement indented under the start of the statement, for consistency.  I'd much rather that the "(SELECT..." went on the line below the EXISTS, if that's necessary to keep the line short (or I would put all the args on the line under g_string_append_printf).

Reply:
I must have overlooked this when I changed it from a count(*) to an EXISTS clause.
I've changed it in 635558d6b5a02a8e99d5cf7edeaf1a6b7525ad22.